brownfield_django/main/views.py

```diff
@@ -135,7 +135,7 @@ def update(self, request, pk=None):
                 student.save()
                 return Response(
                     status=status.HTTP_200_OK)
-            except:
+            except ValueError:
```

Comment:
@sdreher I'm guessing about this one. As I understand .save() could throw a number of different exceptions, and that this isn't the best pattern to catch them.  Should I dig into this more?